docs/orchestration/global-hooks/guides.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Global Hooks Guide
+sidebarTitle: "Guide"
+description: "Follow this step by step guide on creating a global hook in Mage."
+---
+<img
+  alt="Penguin Fishing"
+  src="https://github.com/mage-ai/assets/blob/main/penguin_fishing.gif?raw=True"
+/>
+
+### <b>Step 1: Navigate to the Global Hooks UI</b>
+1. Log in to your Mage account.
+<Note>
+If you are new to Mage follow these steps to enable Global Hooks:
+1. From the Dashboard UI click Pipelines from the left navigation pane
+2. Once in the Pipelines UI click click Settings from the left navigation pane
```

Comment:
nit: comma after "Once in the Pipelines UI"

Reply:
"click" appears twice

Reply:
Added comma, tested in mint dev.